ui/lesson/Terminal.tsx

```diff
@@ -39,40 +30,20 @@ export default function Terminal({
               <span
                 key={index}
                 className={`${
-                  line.type === 'input'
-                    ? 'react-terminal-previous-input text-white/80'
-                    : 'react-terminal-output text-[var(--terminal-output)]'
+                  (line.type === 'answer' &&
+                    'react-terminal-answer rounded bg-green/25 text-green') ||
+                  (line.type === 'output' &&
+                    'react-terminal-output text-[var(--terminal-output)]') ||
+                  (line.type === 'input' &&
+                    'react-terminal-previous-input text-white/80')
```

Comment:
I recognize this code may be a bit convoluted for what it does, and i think it could be simplified somewhat but I hit a blocker on just trying to get it to work without using these operands.